black_nb/cli.py

```diff
@@ -72,6 +77,23 @@
     ),
     show_default=True,
 )
+@click.option(
+    "--extend-exclude",
+    type=str,
```

Comment:
Can you add `callback=black.validate_regex,` to `extend-exclude`, `include` and `exclude`? It returns a `Pattern[str]` type which is needed by `black.get_sources`

Reply:
Absolutely, I have updated these changes.